README.md

```diff
@@ -1,76 +1,35 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://img.shields.io/badge/author1-oracleot-green) ![](https://img.shields.io/badge/author2-IjayAbby-blue)  ![](https://img.shields.io/badge/project-appleclone-orange)
 
-# Project Name
+# Apple Clone
 
-> One paragraph statement about the project.
+> HTML/CSS3 4th Project on Building with Backgrounds and Gradients. 
+> The project focuses on Best CSS practices by learning CSS layouts.
 
-![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+![Screenshot](https://user-images.githubusercontent.com/43843720/88576473-2b421900-d04e-11ea-9594-c42f2fc177db.png)
 
```

Comment:
- [ ] **[OPTIONAL]** You could add the features you put in the message of the pull request in your readme file after your image, to make it more complete if you wish.